iree/tools/BUILD

```diff
@@ -27,12 +27,18 @@ exports_files([
     "sanitizer_suppressions.txt",
 ])
 
-TARGET_COMPILER_BACKENDS = [
+IREE_COMPILER_TARGET_BACKENDS = [
     "//iree/compiler/Dialect/HAL/Target/VMLA",
     "//iree/compiler/Dialect/HAL/Target/VulkanSPIRV",
     "//iree/compiler/Dialect/HAL/Target/LLVM",
 ]
 
+IREE_COMPILER_DIALECT_MODULES = [
```

Comment:
There's an upstream cleanup effort to get rid of variables used in dependencies because they make tool analysis much more difficult. I argued about it, but I think they convinced me that these should very rarely be necessary. They also happen to be a PITA for bazel_to_cmake.

Only commenting because you're adding a new one, but I think we should actually make these sourceless cc_libraries, which achieves the same purpose but is way easier to handle with tooling (including bazel_to_cmake).